src/Illuminate/Database/Eloquent/Factory.php

```diff
@@ -233,7 +233,7 @@ public function offsetGet($offset)
      *
      * @param  string  $offset
      * @param  callable  $value
-     * @return void
+     * @return $this
```

Comment:
This is wrong. ArrayAccess::offsetSet returns void. https://secure.php.net/manual/en/arrayaccess.offsetset.php